index.js

```diff
@@ -1,11 +1,26 @@
 'use strict';
 const fs = require('fs');
 const path = require('path');
+const stream = require('stream');
 const uniqueString = require('unique-string');
 const tempDir = require('temp-dir');
+const isStream = require('is-stream');
+const {writeFile} = fs.promises;
 
 const getPath = () => path.join(tempDir, uniqueString());
 
+const writeStream = async (filePath, data) => new Promise((resolve, reject) => {
+	const writable = fs.createWriteStream(filePath);
+
+	stream.pipeline(data, writable, error => {
```

Comment:
You can await the pipeline.

Reply:
Not without pify.

Reply:
<img width="488" alt="Screenshot 2020-02-24 at 17 06 05" src="https://user-images.githubusercontent.com/170270/75143634-f2f98000-5727-11ea-9043-55cee170c152.png">

Reply:
Fixed!